cms/static/sass/partials/_variables.scss

```diff
@@ -278,3 +278,9 @@ $body-line-height: golden-ratio(.875em, 1);
 // carried over from LMS for xmodules
 $action-primary-active-bg: #1AA1DE; // $m-blue
 $very-light-text: $white;
+
+$color-border: #D9D9D9;
+$color-background: #F2F8FB;
```

Comment:
@andy-armstrong Totally agreed that we should use standard colors. @chris-mike Can we use the existing colors in place of all these new colors?